packages/siopv2-oid4vp-rp-rest-client/src/agent/SiopV2OID4VpRpRestClient.ts

```diff
@@ -31,10 +31,10 @@ export class SiopV2OID4VpRpRestClient implements IAgentPlugin {
     }
   }
 
-  private async deleteDefinitionCorrelation(args: IDeleteDefinitionCorrelationArgs, context: IRequiredContext) {
+  private async deleteDefinitionCorrelation(args: IDeleteDefinitionCorrelationArgs, context: IRequiredContext): Promise<void> {
```

Comment:
You are not deleting a definition or anything. You are deleting the auth session the RP has using the correlation id.

So something more appropropriate would be.

removeAuthRequestSession

Reply:
changed to `removeAuthRequestSession`